index.html

```diff
@@ -0,0 +1,133 @@
+                        <p id="hero-paragraph">The joyful event that addresses the impact of WRC cars,
+                            with guests from all over the world,istake place in April, in Kenya.</p>
+                        <div class="dates">
+                            <h3>2023.04.15(THU) ~ 16(FRI) <span> <br>@Uhuru Gardens,Langata road.Nairobi</span></h3>
+                            <P></P>
+                        </div>
+
+                    </div>
+                    
+            
+                </div>
+
+                <section class="main_program">
+                    <div class="program_top">
+                        <h3>Main Program</h3>   
+                    </div>
+                    <div class="indicator"><span></span></div>
+
+                    <div class="main-program-content">
+                        <div class="program one">
+                            <img src="images/speech.png" alt="speech logo">
+                            <h3>Speech</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <div class="program two">
+                            <img src="images/exhibition.png" alt="speech logo">
+                            <h3>Exhibition</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <div class="program three">
+                            <img src="images/forum-icon.png" alt="speech logo">
+                            <h3>Forum</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <div class="program four">
+                            <img src="images/workshop.png" alt="speech logo">
+                            <h3>Workshop</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <div class="program five">
+                            <img src="images/network-icon.png" alt="speech logo">
+                            <h3>CC Network</h3>
+                            <p>Get a chance to talk to designers and tech founders, ask questions about their journey and inspirations</p>
+                        </div>
+                        <button class="join-btn">Join WRC CAR-NNECT SUMMIT 2023</button>
+                    </div>
+                    
+
+                </section>
+
+                <section class="featured-speakers">
+                   
+
+                </section>
+                <section class="partners" >
+                    <div class="partners_top" id="hide-desktop">
```

Comment:
- [ ] Good job implementing the changes requested by the previous CR 👏 , where he told you that the `partners` and the `foote` sections are not visible. Thanks to your great effort they are visible in the desktop version, however in the mobile version they are not visible. Kindly update that 👍 

![Capture 3](https://user-images.githubusercontent.com/80458199/226099723-dbe56aac-bddf-4efc-8f01-31d5fa1e5b2c.PNG)

Reply:
![Screenshot from 2023-03-18 14-27-22](https://user-images.githubusercontent.com/113665157/226102426-4cd058eb-70ee-4af7-9f89-ed7989c230ab.png)
hey @iambenkis kindly confirm with the original template.They appear in the about section which I have implemented as shown in the attached screenshot.